cmd/juju/secrets/list.go

```diff
@@ -159,6 +162,42 @@ func (c *listSecretsCommand) Run(ctxt *cmd.Context) error {
 	return c.out.Write(ctxt, details)
 }
 
+func humanizeDuration(duration time.Duration) string {
```

Comment:
This is nice, probably be good to extract to a utill package. But I think generally single character suffixes is sufficient.

`1d 5h 25m 59s`  is better than  `1 day 5 hours 25 minutes 59 seconds` which is quiet long.

Reply:
I can move to cmd/juju/common and add next to `UserFriendlyDuration`